Source/Bayonetta_Resolution/rules.txt

```diff
@@ -1,168 +1,168 @@
-#formats =  0x001,0x01a
-tileModesExcluded = 0x001
-overwriteWidth = ($width/$gameWidth) * 640
-overwriteHeight  = ($height/$gameHeight) * 360
-
-#640x352
-
-[TextureRedefine]
-width = 320
-height = 176
-#formats =  0x01a
-overwriteWidth = ($width/$gameWidth) * 320
-overwriteHeight  = ($height/$gameHeight) * 176
-
-#[TextureRedefine] #May unintentionally scale textures
-#width = 256
-#height = 256
-#formatsExcluded =  0x033,0x032,0x031,0x01a,0x001
-#overwriteWidth = ($width/$gameWidth) * 256
-#overwriteHeight = ($height/$gameHeight) * 256
-
-[TextureRedefine]
-width = 160
-height = 80
-#formats = 
-overwriteWidth = ($width/$gameWidth) * 160
-overwriteHeight  = ($height/$gameHeight) * 80
-
-#[TextureRedefine] #May unintentionally scale textures
-#width = 128
-#height = 128
-#formatsExcluded =  0x033,0x031,0x01a,0x001
-#overwriteWidth = ($width/$gameWidth) * 128
-#overwriteHeight = ($height/$gameHeight) * 128
-
-[TextureRedefine]
-width = 96
-height = 48
-#formats = 0x01a
-overwriteWidth = ($width/$gameWidth) * 96
-overwriteHeight  = ($height/$gameHeight) * 48
-
-[TextureRedefine]
-width = 80
-height = 48
-#formats = 
-overwriteWidth = ($width/$gameWidth) * 80
-overwriteHeight  = ($height/$gameHeight) * 48
-
-[TextureRedefine] ##
-width = 64
-height = 32
-#formats = 0x01a
-overwriteWidth = ($width/$gameWidth) * 64
-overwriteHeight  = ($height/$gameHeight) * 32
-
-[TextureRedefine]
-width = 48
-height = 32
-#formats = 
-overwriteWidth = ($width/$gameWidth) * 48
-overwriteHeight  = ($height/$gameHeight) * 32
-
-#[TextureRedefine] #
-#width = 32
-#height = 32
-#formatsExcluded =  0x031,0x002,0x001,0x01a
-#overwriteWidth = ($width/$gameWidth) * 32
-#overwriteHeight = ($height/$gameHeight) * 32
-
-[TextureRedefine]
-width = 32
-height = 16
-#formats = 
-overwriteWidth = ($width/$gameWidth) * 32
-overwriteHeight  = ($height/$gameHeight) * 16
-
-[TextureRedefine] #
-width = 8
-height = 8
-formatsExcluded = 
-overwriteWidth = ($width/$gameWidth) * 8
-overwriteHeight = ($height/$gameHeight) * 8
+[Definition]
+titleIds = 000500001014DB00,0005000010157E00,0005000010157F00
+name = Resolution
+path = "Bayonetta/Graphics/Resolution"
+description = Changes the resolution of the game.
+version = 3
+
+[Preset]
+name = 1280x720 (Default)
+$width = 1280
+$height = 720
+$gameWidth = 1280
+$gameHeight = 720
+
+// Performance
+
+[Preset]
+name = 320x180
+$width = 320
+$height = 180
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 640x360
+$width = 640
+$height = 360
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 960x540
+$width = 960
+$height = 540
+$gameWidth = 1280
+$gameHeight = 720
+
+// Common HD Resolutions
+
+[Preset]
+name = 1920x1080
+$width = 1920
+$height = 1080
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 2560x1440
+$width = 2560
+$height = 1440
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 3200x1800
+$width = 3200
+$height = 1800
+$gameWidth = 1280
+$gameHeight = 720
+
+[Preset]
+name = 3840x2160 4K

```

Comment:
makes stuff a little inconsistent...

Reply:
Oh, I just meant to make a comment out of it, not a review. Ah well. I don't seem to be able to edit the pull request, but it's fine to keep it as is.